functions/src/get-reading/purple-air-response.ts

```diff
@@ -96,6 +140,14 @@ function getReading(
           timestamp = new Date(value * 1000); // eslint-disable-line no-magic-numbers
         }
         break;
+      case 'channel_flags':
+        if (typeof value === 'number') {
+          const channelDownErrors = getChannelFlag(value, channelFlagNames);
+
+          // Add the channel down errors to the overall sensor errors
+          channelDownErrors.forEach(error => (sensorErrors[error] = true));
+        }
+        break;
```

Comment:
I'm a little lost on what `channel_flags` is. Is it a number? A string array? I don't fully understand what's going on w/ this variable.

Reply:
channel_flags is a number that's an index in the channelFlagNames array returned by the PurpleAir API. I'll add a comment explaining this to the docstring of `getChannelFlag`